### PR TITLE
Angular2 Quickstart - edits to first 4 docs, and includes

### PR DESCRIPTION
**Pull request by Catografix**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/client-platforms/angular2/00-intro.md [hunk 1]
```diff
@@ -40,45 +45,32 @@ Throughout the steps, the seed and samples will have the following directory str
...
-    From [npm](https://npmjs.org):
+- [Lock Widget](https://github.com/auth0/lock) is the default authentication widget provided by Auth0.
 
-    ```sh
-    npm install --save auth0-lock
-    ```
+  From [npm](https://npmjs.org):
 
-    From [bower](http://bower.io):
+  `npm install --save auth0-lock`
```
Comment (Catografix) [inline articles/client-platforms/angular2/00-intro.md:55]: for single lines of code, I prefer the inline code tags.

File: articles/_includes/_new_app.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+  <p>Get your credentials (Domain, Client ID, and Client Secret) from the <a href="${uiURL}/#/">dashboard</a>. The sample is pre-configured with your <strong>Default App</strong> credentials.</p>
 <% } else { %>
-  <p>Create an <a href="${uiURL}/login">Auth0 account</a> (or login) and add an authentication application instance from the <a href="${uiURL}/#/">dashboard</a>. Once you create an app, you will be provided with credentials (Domain, Client ID, and Client Secret) which should be stored somewhere safe (do not commit this information to your git repo!). You can download the sample after you login and it will be configured with your Default App credentials.</p>
+  <p>Create an <a href="${uiURL}/login">Auth0 account</a> (or login) and add an authentication client instance from the <a href="${uiURL}/#/">dashboard</a>. Once you create your client, you will be provided with credentials (Domain, Client ID, and Client Secret) which should be stored somewhere safe (do not commit this information to your git repo!). You can download the sample after you login and it will be pre-configured with your Default App credentials.</p>
```
Comment (Catografix) [inline articles/_includes/_new_app.html:4]: Applications are now called Clients. This also makes the screenshots out-of-date.

**Comment by Catografix** [comment]
@chenkie Please review. Would you want me to update the screenshot to show "Clients".

File: articles/client-platforms/angular2/00-intro.md [hunk 3]
```diff
@@ -40,45 +45,32 @@ Throughout the steps, the seed and samples will have the following directory str
...
-```html
-<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/>
-```
+If you are targeting mobile audiences, it's recommended that you add the following in the `<head>` element:
 
-## Done!
+`<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/>`
 
-That's all you need to start working with Auth0!  
-Please continue with [login](/quickstart/spa/angular2/01-login) tutorial to know how to implement basic login.
```
Comment (Catografix) [inline articles/client-platforms/angular2/00-intro.md:84]: Links in this form do not work, they always get redirected to the intro page, as do all refreshes.

File: articles/client-platforms/angular2/01-login.md [hunk 4]
```diff
@@ -7,14 +7,19 @@ description: This tutorial will show you how to integrate Auth0 with angular2 to
...
 
-
 ## Login
 
-The best way to have authentication utilities available across the application is to use an Injectable service. So let's create that and add login functionality there.
+The best way to have authentication utilities available across the application is to use an **Injectable** service. 
+
+You will need an `Auth0Lock` instance to receive your Auth0 credentials and an options object. (For a list of available options, see: [Customization](https://github.com/auth0/lock/tree/v10.0.0#customization).)
+
+Then, add a callback for the `authenticated` lock event, which receives one argument with the login information. When the page is redirected after login, the callback defined for the `authenticated` lock event will be invoked.
```
Comment (chenkie) [inline articles/client-platforms/angular2/01-login.md]: `authenticated` lock event => `authenticated` event

File: articles/client-platforms/angular2/01-login.md [hunk 5]
```diff
@@ -7,14 +7,19 @@ description: This tutorial will show you how to integrate Auth0 with angular2 to
...
 
-We'll need an `Auth0Lock` instance, which receives your Auth0 credentials and an options object (check the available options [here](https://github.com/auth0/lock/tree/v10.0.0#customization))
+For now, just store the `idToken` attribute into `localStorage`.
```
Comment (chenkie) [inline articles/client-platforms/angular2/01-login.md]: Let's avoid using the word 'just'

File: articles/client-platforms/angular2/01-login.md [hunk 6]
```diff
@@ -77,14 +77,8 @@ and then in your component's template
   <button class="btn btn-primary btn-margin" (click)="auth.login()" *ngIf="!auth.authenticated()">Log In</button>
   <button class="btn btn-primary btn-margin" (click)="auth.logout()" *ngIf="auth.authenticated()">Log Out</button>
 </div>
-
 ```
-The lock widget will pop up showing a Login form, when you click the Login button.
 
+The lock widget will display the Login pop-up form when the **Login** button is clicked.
```
Comment (chenkie) [inline articles/client-platforms/angular2/01-login.md]: Let's reword this to: "The Lock widget will be displayed when the **Login** button is clicked.

File: articles/client-platforms/angular2/02-custom-login.md [hunk 7]
```diff
@@ -90,7 +93,7 @@ Now, let's add a form to actually call login:
 
 ## Sign up
 
-To do a Sign Up just call `signup` method on `auth0` instance:
+To allow Sign-up, call the `signup` method on the `auth0` instance:
```
Comment (chenkie) [inline articles/client-platforms/angular2/02-custom-login.md]: To allow Sign-up, call the `signup` method => To allow users to sign up, provide a `signUp` method

File: articles/client-platforms/angular2/02-custom-login.md [hunk 8]
```diff
@@ -117,7 +120,7 @@ and add the button to call this:
 
 ## Social login
 
-To login using social connector, you just need to tell `Auth0` which connection you want to use:
+To login using a social connection, set the `connection` property of the `login` method to the identity provider you want to use:
```
Comment (chenkie) [inline articles/client-platforms/angular2/02-custom-login.md]: To login => To log in

**Comment by Catografix** [comment]
Preview OK: https://auth0-docs-content-pr-1880.herokuapp.com/docs/quickstart/spa/angular2
